sk0-front/src/app/global-services/localstore.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from "@angular/core";
+
+@Injectable({
+  providedIn: "root"
+})
+export class LocalStoreService {
+  constructor() {}
+
+  saveToken(token: string) {
+    localStorage.setItem("token", token);
+  }
+  getToken(): string | null {
+    return localStorage.getItem("token");
+  }
+  removeToken() {
+    localStorage.removeItem("token");
+  }
```

Comment:
You used the `"token"` three different places, would it make sense to create a class level final string variable instead?